src/components/ArticleList.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+import axios from "axios";
+
+const ArticleList = () => {
+const [articles, setArticles] = useState([])
```

Comment:
We should also have a isLoading state here, so we can tell the user if we're still waiting for the data to come back.